CityOfDetroit/bloom
# Remove "Application Due" footer in listings view

# Pull Request Template

## Issue

Addresses #157 


## Description

Removes "Application Due" footer from Listings view.  This update creates a hideApplicationStatus prop for ListingsList to hide the footer when needed.  This change builds on the lessons learned from PR #272. 

![image](https://user-images.githubusercontent.com/82653098/128931278-b5352fa5-cefe-439a-b9aa-e7269f05571d.png)


## Type of change


## How Can This Be Tested/Reviewed?

Check the listings page to see if the footer is present or note.
In Listings.tsx, remove the hideApplicationStatus flag from line 163 to re-enable the "Application Due" footer.


## Checklist:
